### ISSUE TITLE
.ttf files in Windows start menu

### ISSUE THREAD

**Issue by raddyst** [fxsound2/fxsound-app#79]
![image](https://github.com/fxsound2/fxsound-app/assets/8584929/e56dff65-e8d8-4b48-b369-ef8a74c34237)

Windows 7

**Comment by raddyst** [comment]
```
MSI (s) (84:34) [03:17:10:258]: Source for file 'IBMPlexSansArabicRegular.ttf' is compressed
MSI (s) (84:34) [03:17:10:258]: Executing op: SetTargetFolder(Folder=C:\ProgramData\Microsoft\Windows\Start Menu\Programs\FxSound\)
MSI (s) (84:34) [03:17:10:258]: Executing op: SetSourceFolder(Folder=1\SHORTC~1\x86\|SHORTCUTDIR\x86\)
MSI (s) (84:34) [03:17:10:258]: Executing op: FileCopy(SourceName=IBMPLE~1.TTF|IBMPlexSansArabic-Bold.ttf,SourceCabKey=IBMPlexSansArabicBold.ttf_1,DestName=IBMPlexSansArabic-Bold.ttf,Attributes=0,FileSize=236272,PerTick=65536,,VerifyMedia=1,,,,,CheckCRC=0,,,InstallMode=58982400,,,,,,,)
MSI (s) (84:34) [03:17:10:258]: File: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\FxSound\IBMPlexSansArabic-Bold.ttf;	To be installed;	Won't patch;	No existing file
MSI (s) (84:34) [03:17:10:258]: Source for file 'IBMPlexSansArabicBold.ttf_1' is compressed
MSI (s) (84:34) [03:17:11:386]: Executing op: FileCopy(SourceName=IBMPLE~2.TTF|IBMPlexSansArabic-Medium.ttf,SourceCabKey=IBMPlexSansArabicMedium.ttf_1,DestName=IBMPlexSansArabic-Medium.ttf,Attributes=0,FileSize=231888,PerTick=65536,,VerifyMedia=1,,,,,CheckCRC=0,,,InstallMode=58982400,,,,,,,)
MSI (s) (84:34) [03:17:11:386]: File: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\FxSound\IBMPlexSansArabic-Medium.ttf;	To be installed;	Won't patch;	No existing file
MSI (s) (84:34) [03:17:11:386]: Source for file 'IBMPlexSansArabicMedium.ttf_1' is compressed
MSI (s) (84:34) [03:17:11:405]: Executing op: FileCopy(SourceName=IBMPLE~3.TTF|IBMPlexSansArabic-Regular.ttf,SourceCabKey=IBMPlexSansArabicRegular.ttf_1,DestName=IBMPlexSansArabic-Regular.ttf,Attributes=0,FileSize=226004,PerTick=65536,,VerifyMedia=1,,,,,CheckCRC=0,,,InstallMode=58982400,,,,,,,)
MSI (s) (84:34) [03:17:11:406]: File: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\FxSound\IBMPlexSansArabic-Regular.ttf;	To be installed;	Won't patch;	No existing file
MSI (s) (84:34) [03:17:11:406]: Source for file 'IBMPlexSansArabicRegular.ttf_1' is compressed
MSI (s) (84:34) [03:17:11:686]: Executing op: FileCopy(SourceName=NOTOSA~1.TTF|NotoSansJP-Bold.ttf,SourceCabKey=NotoSansJPBold.ttf_1,DestName=NotoSansJP-Bold.ttf,Attributes=0,FileSize=5727828,PerTick=65536,,VerifyMedia=1,,,,,CheckCRC=0,,,InstallMode=58982400,,,,,,,)
MSI (s) (84:34) [03:17:11:686]: File: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\FxSound\NotoSansJP-Bold.ttf;	To be installed;	Won't patch;	No existing file
MSI (s) (84:34) [03:17:11:686]: Source for file 'NotoSansJPBold.ttf_1' is compressed
MSI (s) (84:34) [03:17:12:632]: Executing op: FileCopy(SourceName=NOTOSA~2.TTF|NotoSansJP-Medium.ttf,SourceCabKey=NotoSansJPMedium.ttf_1,DestName=NotoSansJP-Medium.ttf,Attributes=0,FileSize=5729332,PerTick=65536,,VerifyMedia=1,,,,,CheckCRC=0,,,InstallMode=58982400,,,,,,,)
MSI (s) (84:34) [03:17:12:632]: File: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\FxSound\NotoSansJP-Medium.ttf;	To be installed;	Won't patch;	No existing file
MSI (s) (84:34) [03:17:12:632]: Source for file 'NotoSansJPMedium.ttf_1' is compressed
MSI (s) (84:34) [03:17:12:738]: Executing op: FileCopy(SourceName=NOTOSA~3.TTF|NotoSansJP-Regular.ttf,SourceCabKey=NotoSansJPRegular.ttf_1,DestName=NotoSansJP-Regular.ttf,Attributes=0,FileSize=5732824,PerTick=65536,,VerifyMedia=1,,,,,CheckCRC=0,,,InstallMode=58982400,,,,,,,)
MSI (s) (84:34) [03:17:12:738]: File: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\FxSound\NotoSansJP-Regular.ttf;	To be installed;	Won't patch;	No existing file
MSI (s) (84:34) [03:17:12:738]: Source for file 'NotoSansJPRegular.ttf_1' is compressed
MSI (s) (84:34) [03:17:12:838]: Executing op: CacheSizeFlush(,)
MSI (s) (84:34) [03:17:12:838]: Executing op: ActionStart(Name=CreateShortcuts,Description=Creating shortcuts,Template=Shortcut: [1])
```

**Comment by bvijay74** [comment]
Fixed in version 1.1.22.0

**Comment by fai8146767** [comment]
###
